src/guiguts/maintext.py

```diff
@@ -432,8 +434,8 @@ def do_column_select(self, col_range: IndexRange) -> None:
             # If line is too short for any text to be selected, select to
             # beginning of next line (just captures the newline). This
             # is then dealt with in column_copy_cut.
-            if self.get(beg, end) == "":
-                end += "+ 1l linestart"
+            # if self.get(beg, end) == "":
```

Comment:
Leftover debugging or should this be deleted?

Reply:
Deleted - thanks